### DISCUSSION TITLE
萌新，不怎么懂代码，怎么实现目录下方有文本内容

### DISCUSSION THREAD

**Discussion by sail5200919** [AlistGo/alist#1236] [Q&A] [answered]
主要看别的用这个搭建的网盘分享，有这种功能
紧跟目录的下方有另外一个方格里面有标题，超文本，超链接什么的。而且和上面目录的样式是一致的。
纯萌新一个，求大佬指教

**Comment by Pandaft** [comment, marked as answer]
在目录下放一个 `readme.md` 文件，内容语法格式参考 markdown 。

**Reply by sail5200919** [reply]
大佬那个首页改怎么搞，我弄了很多网盘上去，然后首页显示的是各网盘的名字，这也没法上传md文件，这要怎么搞

**Reply by Pandaft** [reply]
一样的道理，添加一个 `账号` ，虚拟路径为 `/` ，该账号的根目录就是首页。

**Comment by ghost** [comment]
在元信息里面添加元信息，路径为/，然后说明里面写内容
